cmake/Modules/FindZlib_EP.cmake

```diff
@@ -83,8 +83,8 @@ if (NOT ZLIB_FOUND)
 
     ExternalProject_Add(ep_zlib
       PREFIX "externals"
-      URL "http://iweb.dl.sourceforge.net/project/libpng/zlib/1.2.11/zlib1211.zip"
-      URL_HASH SHA1=bccd93ad3cee39c3d08eee68d45b3e11910299f2
+      URL "https://downloads.sourceforge.net/project/libpng/zlib/1.2.11/zlib1211.zip"
+      URL_HASH SHA1=b8d230cc5effe6dffa2807e0e342245b0683f2df
```

Comment:
I just downloaded directly from the new URL in browser, and the SHA is unchanged (`bccd...`). If you used curl you might have gotten the HTML redirect page.

Reply:
Corrected